SPECS/gpumon.spec

```diff
@@ -49,6 +49,9 @@ DESTDIR=%{buildroot} %{__make} install
 %{_unitdir}/xcp-rrdd-gpumon.service
 
 %changelog
+* Fri Jul 21 2023 Benjamin Reis <benjamin.reis@vates.fr> - 0.24.0-8.2
+- Bump release and rebuild for xs-opam-repo-6.66.0-1.2.xcpng8.3 and xapi-23.3.0-1.6.xcpng8.3
```

Comment:
It's usually enough to say "Rebuild". The bumped release is implied.

It would be good also if the commit message would contain the same level of detail as the changelog: "Rebuild for xs-opam-repo-6.66.0-1.2.xcpng8.3 and xapi-23.3.0-1.6.xcpng8.3"

Reply:
Yeah, I'll try to get it documented at some point, but I don't think we need to specific "Bump release" and I usually don't. I find it more important to describe why we rebuild, as you've done.